src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php

```diff
@@ -94,24 +94,25 @@ public function __construct($inputTimezone = null, $outputTimezone = null, $form
      *                                       instance or if the output timezone
      *                                       is not supported.
      */
-    public function transform($value)
+    public function transform($dateTime)
```

Comment:
I suggest to avoid renaming the variable. It will make merging between branches much easier